docs/.vitepress/config.mts

```diff
@@ -0,0 +1,85 @@
+import { defineConfig } from 'vitepress'
+
+// https://vitepress.dev/reference/site-config
+export default defineConfig({
+  title: "CanCanCan",
+  description: "The authorization Gem for Ruby on Rails.",
+  head: [
+    ['link', { rel: "apple-touch-icon", sizes: "180x180", href: "/apple-touch-icon.png" }],
+    ['link', { rel: "icon", type: "image/png", sizes: "32x32", href: "/favicon-32x32.png" }],
+    ['link', { rel: "icon", type: "image/png", sizes: "16x16", href: "/favicon-16x16.png" }],
+    ['link', { rel: "mask-icon", href: "./safari-pinned-tab.svg", color: "#3c3ebf" }],
+    ['link', { rel: "manifest", href: "/manifest.json" }],
+
+    ['link', { rel: "icon", href: "/favicon.ico", type: "image/x-icon" }],
+    ['link', { rel: "shortcut icon", href: "/favicon.ico", type: "image/x-icon" }],
+  ],
+  sitemap: {
+    hostname: 'https://cancancan.com'
```

Comment:
assuming this will be the domain

Reply:
or maybe cancancan.dev is better?

Reply:
this will not only boost the SEO but also helps the user when he searches something on any page since it now gets indexed by google and so